contributors/devel/development.md

```diff
@@ -139,7 +139,7 @@ the Windows Command Prompt.
 
 1. If you're using Windows 10, Version 2004, Build 19041 or higher,
    you can use Windows Subsystem for Linux (WSL) to build
-   Kubernetes. [Follow these instructions to install WSL2.](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
+   Kubernetes. [Follow these instructions to install WSL2.](https://docs.microsoft.com/en-us/windows/wsl/install-win10). After that we need to install Ubuntu on Windows Subsystem for Linux (WSL) · Enable WSL on Windows 10. [Follow these instructions to install Ubuntu and enabling wsl2.](https://ubuntu.com/wsl). Now you should clone the repo inside ubuntu and you are ready.
```

Comment:
Thanks for you PR, but I double checked the wsl instructions that are linked and they do tell you to download and install your preferred linux distro as part of step 6. The default simplified install will install Ubuntu as well. I don't think we actually need to call this explicitly, and we don't want to bias people to installing one distro over another (e.g. it works with suse etc listed on there too).

Reply:
Yeah, you are right. Thanks for the review.